File: VB-Code/Form1/handlers/Pertablebill.vb
Public Class Pertablebill
    ' Dim tableNo As Integer
    Dim issaved As Boolean = False
    Dim totalwithouttax As Decimal
    Dim tax As Decimal = CDec(0)
    Dim tatalwithtax As Decimal = CDec(0)
    Dim discount As String = ""

    Private _bill As List(Of Aclass) = New List(Of Aclass)
    Public Property Bill() As List(Of Aclass)
        Get
            Return _bill
        End Get
        Set(ByVal value As List(Of Aclass))
            _bill = value
        End Set
    End Property



    Private _tableNo As Integer
    Public Property tableNo As Integer
        Get
            Return _tableNo
        End Get
        Set(ByVal value As Integer)
            _tableNo = value
        End Set
    End Property


End Class

File: C#-Code/Demo.Hotel/handlers/Pertablebill.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Demo.Hotel.handlers
{
    class Pertablebill
    {
        bool issaved = false;
        decimal totalwithouttax;
        decimal tax = Convert.ToDecimal(0);
        decimal tatalwithtax = Convert.ToDecimal(0);
        string discount = "";


        public int _tableNo { get; set; }

        public List<Aclass> _bill = new List<Aclass>();
        //    public property List<Aclass>  Bill() As List(Of Aclass)
        //    Get
        //        Return _bill
        //    End Get
        //    Set(ByVal value As List(Of Aclass))
        //        _bill = value
        //    End Set
        //End Property
    }
}
